dwave/cloud/solver.py

```diff
@@ -1174,6 +1174,212 @@ def check_problem(self, linear, quadratic):
 
         return self.nodes.issuperset(linear) and self.edges.issuperset(quadratic)
 
+    def estimate_qpu_access_time(self,
+                                 num_qubits: int,
+                                 num_reads: int = 1,
+                                 annealing_time: typing.Optional[float] = None,
+                                 anneal_schedule: typing.Optional[typing.List[typing.Tuple[float, float]]] = None,
+                                 initial_state:  typing.Optional[typing.List[typing.Tuple[float, float]]] = None,
+                                 reverse_anneal: bool = False,
+                                 reinitialize_state: bool = False,
+                                 programming_thermalization: typing.Optional[float] = None,
+                                 readout_thermalization: typing.Optional[float] = None,
+                                 reduce_intersample_correlation: bool = False,
+                                 **kwargs) -> float:
+        """Estimates QPU access time for a submission to the selected solver.
+
+        Estimates a problem’s quantum processing unit (QPU) access time from the
+        parameter values you specify, timing data provided in the ``problem_timing_data``
+        `solver property <https://docs.dwavesys.com/docs/latest/c_solver_properties.html>`_,
+        and the number of qubits used to embed the problem on the selected QPU, as
+        described in the
+        `system documentation <https://docs.dwavesys.com/docs/latest/c_qpu_timing.html>`_.
+
+        Requires that you provide the number of qubits to be used for your
+        problem submission. :term:`Embedding` is typically heuristic and the number
+        of required qubits can vary between executions. If you are using a heuristic
+        embedding tool such as
+        :std:doc:`minorminer <oceandocs:docs_minorminer/source/sdk_index>` indirectly
+        through your sampler (e.g., by using the
+        :class:`~dwave.system.composites.EmbeddingComposite` or :class:`~dwave.system.samplers.DWaveCliqueSampler` classes), you can use
+        the same tool on your problem to estimate the expected number of qubits:
+        for large, complex problems you might run the tool several times and take
+        the number of qubits from the produced average or worst-case embedding;
+        for small, simple problems even a single run might be sufficient. If you
+        are using such a tool directly (e.g., in conjunction with the
+        :class:`~dwave.system.composites.FixedEmbeddingComposite`)
+        or otherwise generating a heuristic or non-heuristic embedding, take
+        the required number of qubits from your embedding. Because embedding
+        depends on a QPU’s working graph, such embeddings should be for
+        the particular QPU for which you are estimating the access time.
```

Comment:
Given this method is in cloud-client now, where all problems are already embedded, perhaps we can simplify the embedding discussion?